test/espnet2/enh/nets/test_beamformer_net.py

```diff
@@ -111,28 +105,27 @@ def test_beamformer_net_forward_backward(
     [(8, None, 2)],
 )
 @pytest.mark.parametrize("num_spk", [1, 2])
-@pytest.mark.parametrize("normalize_input", [True, False])
+@pytest.mark.parametrize("normalize_input", [True])
 @pytest.mark.parametrize("mask_type", ["IPM^2"])
 @pytest.mark.parametrize("loss_type", ["mask_mse", "spectrum"])
 @pytest.mark.parametrize("use_wpe", [False])
-@pytest.mark.parametrize("wnet_type", ["blstmp"])
-@pytest.mark.parametrize("wlayers", [3])
-@pytest.mark.parametrize("wunits", [8])
-@pytest.mark.parametrize("wprojs", [10])
-@pytest.mark.parametrize("wdropout_rate", [0.0, 0.2])
+@pytest.mark.parametrize("wnet_type", ["lstm"])
+@pytest.mark.parametrize("wlayers", [2])
+@pytest.mark.parametrize("wunits", [2])
+@pytest.mark.parametrize("wprojs", [2])
+@pytest.mark.parametrize("dropout_rate", [0.0, 0.2])
```

Comment:
The dropout_rate is useless with `model.eval()`, so Line 116 can be modified to:
```
@pytest.mark.parametrize("dropout_rate", [0.0])
```